packages/did-eth-registry/contracts/DIDRegistry.sol

```diff
@@ -269,4 +279,115 @@ contract EthereumDIDRegistry {
         );
         revokeAttribute(identity, checkSignature(identity, sigV, sigR, sigS, digest), name, value);
     }
+
+    /**
+     * @dev check signature of the owner
+     * @param identity address of the identity
+     * @param sigV signature V
+     * @param sigR signature R
+     * @param sigS signature S
+     * @param digest digest of the message
+     */
+    function checkSignature(
```

Comment:
I think it might be better to prefix these internal functions with underscores at some point. What do you think?

Reply:
Yes, that's a nice idiom to prevent namespace pollution.  I'll push that change in a few minutes